### ISSUE TITLE
How to finetune inpainting model for object removal? What is the input prompt for object removal for both training and inference?

### ISSUE THREAD

**Issue by hdjsjyl** [huggingface/diffusers#1921]
Hi team,
Thanks for your great work!
I am trying to get object removal functionality from inpainting of SD.
How to finetune inpainting model for object removal?
What is the input prompt for object removal for both training and inference?
Thanks

**Comment by patrickvonplaten** [comment]
Hey @hdjsjyl,

Here also a related issue: https://github.com/huggingface/diffusers/issues/973
Also slightly related: https://huggingface.co/spaces/afmck/stable-diffusion-inpainting-segmentation


So to directly answer some questions:

1. [There is probably some good research on this] Intuitively what should work would be to take some images without objects + prompt: "e.g. a nice mountain landscape", then add an object, such as a cow and put it over the landscape image, then mask the cow away and predict the original image with the original slogan.
2. & 3. Prompt for object removal during inference is simply a description of the image **without** the object as stated in #973

**Comment by shileims** [comment]
Hi @patrickvonplaten ,
     Thanks for your reply and sorry for the late response.
     For 1,2,3, I would like to fix the input prompt. Without inputting a different prompt, with a fixed input prompt, I hope this model performs object removal functionality, not text-guided object removal functionality.

**Comment by manurare** [comment]
Hey @shileims . Did you get any progress fine-tuning inpainting without prompt?
